rollup.config.js

```diff
@@ -9,13 +9,16 @@ import json from '@rollup/plugin-json';
 import copy from 'rollup-plugin-copy';
 import del from 'rollup-plugin-delete';
 import webpackStatsPlugin from 'rollup-plugin-webpack-stats';
+import svelte from 'rollup-plugin-svelte';
+import {sveltePreprocess} from 'svelte-preprocess';
```

Comment:
The cool part is that this PR finally adds support for proper dynamic components that can even be used via our native-DOM JSX solution. This will eventually help features on React views, for example:

- https://github.com/refined-github/refined-github/issues/6554

Related:

- https://github.com/refined-github/refined-github/issues/6493
- https://github.com/refined-github/refined-github/issues/4058